model/project_parser.go

```diff
@@ -567,15 +568,15 @@ func LoadProjectInto(ctx context.Context, data []byte, opts GetProjectOpts, iden
 		opts.RemotePath = path.FileName
 		yaml, err := retrieveFile(ctx, opts)
 		if err != nil {
-			return nil, errors.Wrapf(err, LoadProjectError)
+			return intermediateProject, errors.Wrapf(err, LoadProjectError)
```

Comment:
Actually sorry, can you add a comment at the top about why we return intermediateProject as it is, even in error? (because we may want to see how far we got with the merging process before we ran into an issue) This behavior is non-standard so it's good for future engineers to know that this was a purposeful choice.

Reply:
done